tests/phpunit/Unit/MarksimpleTest.php

```diff
@@ -14,6 +14,27 @@
 class MarksimpleTest extends AbstractTestCase
 {
 
+    /**
+     * The unreaddable file path.
+     *
+     * @var string
+     */
+    protected $noReadableFile = __DIR__ . '/./no_readable_file.txt';
+
+    public function setUp()
+    {
+        touch($this->noReadableFile);
+        chmod($this->noReadableFile, 27);
+    }
+
+    public function tearDown()
```

Comment:
The `setUp` and `tearDown` methods are about the PHPUnit method.

I think these methods should not add the `PHPDoc` comment annotations.

Reply:
Thanks a lot for work on, all is fine, the today's and the reason for the phpdoc on the unit tests functions, maybe inherit as phpdoc key?

Reply:
Thank you for your reply. Agreed because of consistency.

How about adding the comment annotations for these two methods?

Reply:
I add normally a simple doc part, like this https://github.com/bueltge/adminimize/blob/refactor/tests/phpunit/Unit/AbstractTestCase.php

Reply:
Have you seen the last check https://scrutinizer-ci.com/g/bueltge/marksimple/inspections/d1df82df-1493-429c-9b40-424882755e0f
I think only 1 parameter is possible?

Reply:
Hi @bueltge, thank you for your reply.

I think the `log` method should be called via `$this->logger` class instance.